developers/blobstreamx-offchain.md

```diff
@@ -1,18 +1,18 @@
 ---
-description: Learn how to integrate your L2's offchain logic with Blobstream
+description: Learn how to integrate your L2's offchain logic with Blobstream X
 ---
 
-# Integrate with Blobstream client
+# Integrate with Blobstream X client
 
-## Blobstream demo rollup
+## BlobstreamX demo rollup
```

Comment:
nit: Blobstream X, I think we use a space everywhere

Reply:
[d7d2f14](https://github.com/celestiaorg/docs/pull/1379/commits/d7d2f14e872ee4fcc5a06079a3e5b0c29840e0f2)